source/hardware/controllers/native_instruments_traktor_kontrol_s3.rst

```diff
@@ -0,0 +1,161 @@
+Native Instruments Traktor Kontrol S3
+=====================================
+
+.. versionadded:: 2.3.0
+
+.. figure:: ../../_static/controllers/native_instruments_traktor_kontrol_s3.jpg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Native Instruments Traktor Kontrol S3 (top view)
+   :figclass: pretty-figures
+
+   Native Instruments Traktor Kontrol S3 (top view)
+
+References:
+
+  - `Manufacturer's product
+    page <https://www.native-instruments.com/en/products/traktor/dj-controllers/traktor-kontrol-s3/>`__
+  - `DJTechTools
+    review <https://djtechtools.com/2019/10/16/traktor-kontrol-s3-the-middle-child-for-the-mk3-traktor-generation/>`__
+  - `DJWORX
+    overview <https://djworx.com/the-traktor-kontrol-s3-we-have-it-but-not-the-software/>`__
```

Comment:
```suggestion
  - `Manufacturer's product
    page <https://www.native-instruments.com/en/products/traktor/dj-controllers/traktor-kontrol-s3/>`__
```

I don't think review links should go into the manual. You can put them on a wiki page if you want.

Reply:
moving the reference to the bottom since it's not really important